config/jobs/kubernetes/sig-arch/kubernetes-depstat-periodical.yaml

```diff
@@ -21,11 +21,8 @@ periodics:
           pushd "$ARTIFACTS"; \
           go install github.com/kubernetes-sigs/depstat@latest; \
           popd; \
-          depstat stats --json > "${ARTIFACTS}/stats.json"; \
```

Comment:
we should print the output and store it into a file as well. will be useful for later automation, look at the `tee` command in linux